include/exadg/fluid_structure_interaction/precice/driver_fluid.h

```diff
@@ -218,7 +216,32 @@ class DriverFluid : public Driver<dim, Number>
 
       // post-solve
       if(is_new_time_window)
+      {
+        // computes new time-step size
         fluid->time_integrator->advance_one_timestep_post_solve();
+        // next, we synchronize the time-step sizes. Have a look in the base class for more
+        // explanations. Subcycling would in theory be compatible in explicit coupling schemes here.
+        // In implicit coupling schemes, we need matching time-window sizes as the time-step size
+        // push back happens in the 'advance_one_timestep_post_solve()' and we cannot change two
+        // subsequent time-step sizes without a push back operation, otherwise we falsify the time
+        // integrator. However, the treatment of the boundary conditions is not handled correctly
```

Comment:
I do not understand this sentence.

Reply:
I'm referring to this line 
https://github.com/exadg/exadg/blob/6e647f226111b184e441eee8d7b8326e48bc757f/include/exadg/time_integration/time_int_bdf_base.cpp#L242  and this nice comment https://github.com/exadg/exadg/blob/6e647f226111b184e441eee8d7b8326e48bc757f/include/exadg/time_integration/time_int_bdf_base.h#L90-L94

Does it make it more clear?